h5io/_h5io.py

```diff
@@ -333,6 +348,56 @@ def _triage_write(
                 cleanup_data.append({title: (rootname, key, value)})
                 return
 
+        if use_state:
+            class_type = str(type(value)).split("'")[1]
```

Comment:
I would have thought this maybe more robust/standard than relying on the `__repr__` being `<class 'whatever'>`?
```suggestion
            class_type = value.__class__.__name__
```

Reply:
I used `value.__class__.__module__ + "." + value.__class__.__name__` to get both the module class and its name.